### DISCUSSION TITLE
Episode 187: Parallel Top N

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#86] [Episode]
Episode Link: https://adspthepodcast.com/2024/06/21/Episode-187.html

Feel free to write comments or ask questions 🙂

**Comment by jamie23** [comment]
Great episode as per usual @codereport ! Just a heads up though, we pronounce Tomato 'Tom ah toe' over here in the UK. When I was listening I thought you were about to say nobody says 'tom-ay-toe' 😄

**Reply by codereport** [reply]
oops, my bad :laughing: will try to remember to mention this on the next recording : )

**Comment by SKyletoft** [comment]
> Tomaytoe, tomato

That is a perfectly reasonable phrase. ~~Sensible people~~ _brits_ say tomahtoe.

The nonsensical version is how it has somehow morphed into potaytoe, potahtoe because noone says potahtoe.
